diff -urN gmpc-extraplaylist-0.15.5.0/src/Makefile.am gmpc-extraplaylist-0.15.5.0.new/src/Makefile.am
--- gmpc-extraplaylist-0.15.5.0/src/Makefile.am	2007-11-29 14:54:06.000000000 +0100
+++ gmpc-extraplaylist-0.15.5.0.new/src/Makefile.am	2008-02-04 10:16:12.158251014 +0100
@@ -1,6 +1,6 @@
-libdir=${prefix}/share/gmpc/plugins/
+pluginsdir=$(libdir)/gmpc/
 
-lib_LTLIBRARIES = libextraplaylist.la
+plugins_LTLIBRARIES = libextraplaylist.la
 
 libextraplaylist_la_SOURCES=\
 	main.c
